packages/insomnia-inso/src/logger.ts

```diff
@@ -0,0 +1,43 @@
+import consola, { LogLevel, FancyReporter, BasicReporter, logType } from 'consola';
+import { noop } from './util';
+
+export const noConsoleLog = async <T>(callback: () => Promise<T>): Promise<T> => {
+  const oldConsoleLog = console.log;
+
+  console.log = noop;
+
+  try {
+    return await callback();
+  } finally {
+    console.log = oldConsoleLog;
+  }
+};
+
+export type LogsByType = {
+  [t in logType]?: string[]
+}
+
+export type ModifiedConsola = ReturnType<typeof consola.create> & { __getLogs: () => LogsByType};
+
+const consolaLogger = consola.create({
+  reporters: [
+    new FancyReporter({
+      formatOptions: {
+        // @ts-expect-error something is wrong here, ultimately these types come from https://nodejs.org/api/util.html#util_util_inspect_object_options and `date` doesn't appear to be one of the options.
```

Comment:
🤔

Reply:
yep.  I didn't want to change it though because I'm afraid of unknown changes and I didn't know how to test this so I left it here for now.  if you think it's safe to remove please let me know but I didn't want to take the chance right now (or, as with any "right now", perhaps, ever).

Reply:
@reynolek @dimitropoulos these options come from https://github.com/unjs/consola/blob/4477e87a5153d31d22a2692565959d90599a6b61/src/reporters/fancy.js#L9-L16, that link in the comment seems strange 🤔